### ISSUE TITLE
Dynmap Empty when zoomed out.

### ISSUE THREAD

**Issue by IngeniousCoder** [webbukkit/dynmap#4109]
**Issue Description:** Dynmap is EMPTY at default zoom despite fullrender and world generation completed

* **Dynmap Version:** 3.7-beta-5
* **Server Version:** paper 1.20.2
* **Other Relevant Data/Screenshots:**
![image](https://github.com/webbukkit/dynmap/assets/29097583/472506a4-60e3-415d-a792-1505859c7535)
Dynmap when loaded into the site.
![image](https://github.com/webbukkit/dynmap/assets/29097583/fb474776-bed5-44de-adc9-6e79486d582b)
Dynmap beginning to render once zooming in
![image](https://github.com/webbukkit/dynmap/assets/29097583/d7ffe1f8-6940-4b97-aea4-261355e50e13)
Dynmap partially loading at certain zoom
![image](https://github.com/webbukkit/dynmap/assets/29097583/ec0a044c-7a4e-4b88-9002-029333bc3c4b)
Dynmap only loads after zooming even further

* **Steps to Replicate:** Literally just start a new fullrender of any world.

 [x] *I have looked at all other issues and this is not a duplicate*  
 [x] *I have been able to replicate this*

**Comment by JurgenKuyper** [comment]
purge the map and fullrender it without restarting the server @IngeniousCoder

**Comment by IngeniousCoder** [comment]
Done it at least thrice. Same issue.

**Comment by JurgenKuyper** [comment]
did you purge, wait on finish and a new fullrender? not just a fullrender with existing tiles?

**Comment by IngeniousCoder** [comment]
Yes, purged. Manually verified all files gone in filesystem before fullrender. 

Still error.

**Comment by JurgenKuyper** [comment]
then you are using a domain proxy that is caching the tiles, which might be wrong

**Comment by IngeniousCoder** [comment]
I am validating from 127.0.0.1:8123, same issue.

**Comment by VL4DST3R** [comment]
This is a [long standing issue](https://github.com/webbukkit/dynmap/issues/3182) on virtually all plugin and game versions that seems to never be addressed, so I wouldn't hold my breath. I've been lurking here in hopes for a fix for a long time now and nothing reliable has been provided, [although there was apparently an attempt?](https://github.com/webbukkit/dynmap/issues/4014) just "do it again till it works" level fixes, which obviously is not feasible when it takes 2 days to render a map.

**Comment by IngeniousCoder** [comment]
Feels like RNG, if it happens to a server it happens, if it doesn't then it doesn't.

**Comment by VL4DST3R** [comment]
> Feels like RNG, if it happens to a server it happens, if it doesn't then it doesn't.

For the longest time I assumed it is due to paper and its async stuff, but I've seen people mentioning it happening on bukkit as well, so I don't think there's much you can do.

This wouldn't be an issue if we had _actual_ control over how LODs gen: define its speed, proper (i.e. working) validation via something like `initial-zoomout-validate` or at least a way to force _only_ LODs to regen based on the pre-existing "LOD0" tiles, since it realistically should take a lot less compared to a fullrender.

**Comment by patrix87** [comment]
Same issue here on 1.20.1 Dynamp 3.7-beta-6.
All tiles rendered when zoomed it.
Nothing when zoomed out.
It worked fine before I set /dmap worldset world extrazoomout:3
It did a purgeworld then a fullrender after that and nothing.
